apps/nextjs-website/src/_contents/pagoPa/guides.ts

```diff
@@ -40,6 +40,10 @@ const sanp: GuideDefinition = {
     slug: 'sanp',
   },
   versions: [
+    {
+      version: '3.6.1',
+      dirName: 'PiBduZS70S7Ae4WpWywl',
+    },
```

Comment:
@certevol Since you added a new version, then please add the changeset as well 🙏 This will be a new `minor` release.